### PR TITLE
PF-2543: list workspaces will include missing authdomains 

### PR DESCRIPTION
**Pull request by yuhuyoyo**

### CODE REVIEW

File: openapi/src/parts/workspace.yaml [hunk 1]
```diff
@@ -659,6 +659,11 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/WsmPolicyInput'
+        missingAuthDomains:
+          type: array
+          description: The auth domains that the user is missing for this workspace
```
Comment (ddietterich) [inline openapi/src/parts/workspace.yaml:664]: Maybe check with @cbookg about what he'd like to call it? In Sam-land it is an auth domain, but in policy-land it is a failed group constraint. I don't feel strongly about what we call it here.

**Review by ddietterich — APPROVED** [review]
Nice!

**Review by mikenev — APPROVED** [review]
lgtm

File: openapi/src/parts/workspace.yaml (hunk 1 above)
Comment (yuhuyoyo) [reply openapi/src/parts/workspace.yaml:664]: chris approved the field in the ticket comment

**Pull request merged**